### PR TITLE
Fixup Shine

### PR DESCRIPTION
**Pull request by CodaFi**
An apology for #25, if you will.

### CODE REVIEW

**Comment by CodaFi** [comment]
⛵️

**Comment by harlanhaskins** [comment]
🙌

File: Sources/Crust/Shine.swift [hunk 1]
```diff
@@ -18,38 +18,99 @@ private enum LayoutBlock {
...
         continue
-      // Keep searching
-      default:
+      case .comment(_):
         continue
+      default:
+        newl = true
+        break
+      }
+      break
```
Comment (harlanhaskins) [inline Sources/Crust/Shine.swift:59]: Is this correct?

Comment (CodaFi) [reply Sources/Crust/Shine.swift:59]: It iterates in reverse until the nearest non-tab or non-space and records that there was a newline. Odd, but correct.

Comment (harlanhaskins) [reply Sources/Crust/Shine.swift:59]: Ah I wasn't considering the `continue`s. This is fine.

**Pull request merged**